gradle.properties

```diff
@@ -1,5 +1,5 @@
 mcversion=1.12.2
-forgeversion=14.23.2.2637
+forgeversion=14.23.3.2655
```

Comment:
This is the forge version used in development. 
It's good to update this, but we also need to update the minimum required version here: https://github.com/ForestryMC/ForestryMC/blob/f52cc3893b2797c7e25761fc24f7ed2e6859c561/src/main/java/forestry/Forestry.java#L71

Look through the [forge changelog here](http://files.minecraftforge.net/maven/net/minecraftforge/forge/1.12.2-14.23.3.2676/forge-1.12.2-14.23.3.2676-changelog.txt) to find which version adds the oredict entries, and use that version as the minimum required.